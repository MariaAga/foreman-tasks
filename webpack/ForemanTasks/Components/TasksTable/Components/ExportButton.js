import React from 'react';
import { Button } from 'patternfly-react';
import PropTypes from 'prop-types';
import { translate as __ } from 'foremanReact/common/I18n';

export const ExportButton = ({ url }) => (
  <Button className="export-csv" href={url} title={__('Export to CSV')}>
    {__('Export')}
  </Button>
);

ExportButton.propTypes = {
  url: PropTypes.string.isRequired,
};
