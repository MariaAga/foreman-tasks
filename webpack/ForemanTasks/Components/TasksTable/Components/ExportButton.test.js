import { testComponentSnapshotsWithFixtures } from 'react-redux-test-utils';

import { ExportButton } from './ExportButton';

const fixtures = {
  'render with minimal props': { url: 'url' },
};

describe('ExportButton', () =>
  testComponentSnapshotsWithFixtures(ExportButton, fixtures));
