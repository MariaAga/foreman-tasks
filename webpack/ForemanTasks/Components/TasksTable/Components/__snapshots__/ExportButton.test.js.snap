// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExportButton render with minimal props 1`] = `
<Button
  active={false}
  block={false}
  bsClass="btn"
  bsStyle="default"
  className="export-csv"
  data-no-turbolink="true"
  disabled={false}
  href="url"
  title="Export to CSV"
>
  Export
</Button>
`;
